src/hotspot/share/gc/z/zPhysicalMemory.cpp

```diff
@@ -173,6 +173,10 @@ void ZPhysicalMemoryManager::free(const ZPhysicalMemory& pmem) {
   _backing.free(pmem);
 }
 
+void ZPhysicalMemoryManager::pretouch(uintptr_t offset, size_t size) const {
+  _backing.pretouch(offset, size);
```

Comment:
pretouch is not declared or defined in bsd/windows, so compilation on those platforms will fail. Is it better to add an empty implementation on those platforms (with ShouldNotReach)? A compilation error looks worse than runtime error msg.

Reply:
Windows does not compile ZGC. Our daily pipeline can verify it. No worries.